.travis.yml

```diff
@@ -22,8 +22,7 @@ before_script:
  - composer install --no-interaction
 
 after_script:
-  - wget https://scrutinizer-ci.com/ocular.phar
-  - php ocular.phar code-coverage:upload --format=php-clover coverage.clover
+  - if [ $TRAVIS_PHP_VERSION = '5.6' ]; then wget https://scrutinizer-ci.com/ocular.phar; php ocular.phar code-coverage:upload --format=php-clover coverage.clover; fi
```

Comment:
Not sure which PHP version you'd like coverage of, so I used 5.6 which is as good as any ^_^